common/test/services/IndexPageTest.scala

```diff
@@ -48,20 +48,16 @@ import scala.concurrent.Future
         fail("Wrong type (expected: IndexPage, real: Result)")
       case Some(page) =>
         val front = IndexPage.makeFront(page, edition)
-        front.containers.length should be(1)
+        front.containers should not be empty
```

Comment:
@jfsoul @NataliaLKB I've refactored this test because it shouldn't fail as a result of changing the CAPI dependency version.